.travis.yml

```diff
@@ -29,5 +30,6 @@ rvm:
   - "2.0"
   - "2.1"
   - "2.2"
+  - "2.3"
```

Comment:
FWIW: didn't got it working like this; had to replace it with ```- 2.3.0``` to have it working.